src/main/kotlin/no/nav/familie/tilbake/behandling/FagsakMapper.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.familie.tilbake.behandling
+
+import no.nav.familie.kontrakter.felles.PersonIdent
+import no.nav.familie.tilbake.api.dto.BehandlingsoppsummeringDto
+import no.nav.familie.tilbake.api.dto.BrukerDto
+import no.nav.familie.tilbake.api.dto.FagsakDto
+import no.nav.familie.tilbake.behandling.domain.Behandling
+import no.nav.familie.tilbake.behandling.domain.Fagsak
+import no.nav.familie.tilbake.integration.pdl.internal.PersonInfo
+
+object FagsakMapper {
+
+    fun tilRespons(fagsak: Fagsak,
+                   personInfo: PersonInfo,
+                   behandlinger: List<Behandling>): FagsakDto {
+        val bruker = BrukerDto(personIdent = PersonIdent(fagsak.bruker.ident),
+                               navn = personInfo.navn,
+                               fødselsdato = personInfo.fødselsdato,
+                               kjønn = personInfo.kjønn)
+
+        val behandlingListe = behandlinger.map { behandling ->
```

Comment:
Jeg vet det kan være uvant, når man ikke har brukt Kotlin en stund, men bruk "it"  i lambda-uttrykk i setdet for navngitt parameter. Det er lettere for de fleste å lese.